crates/hir_ty/src/tests/method_resolution.rs

```diff
@@ -108,16 +108,16 @@ fn infer_associated_method_with_modules() {
     check_infer(
         r#"
         mod a {
-            struct A;
+            pub struct A;
```

Comment:
there should be no changes to the `hir_ty` tests necessary at all :slightly_smiling_face: